rct229/ruleset_functions/get_baseline_system_types.py

```diff
@@ -87,4 +88,25 @@ def get_baseline_system_types(rmi_b):
                 baseline_hvac_system_dict[hvac_sys].append(hvac_b_id)
                 continue
 
+            # Add error handling
```

Comment:
This is a really clever use of iterators!
As much as I do like it, I think it might be more straight forward to get a count of the hvac systems:
```
hvac_sys_count = len([
    hvac_b_id 
    for hvac_sys_key in baseline_hvac_system_dict.keys()
    for hvac_b_id in baseline_hvac_system_dict[hvac_sys_key]
])
if hvac_sys_count == 0:
     raise ...
elif hvac_sys_count > 1:
    raise ...

Reply:
I think you need a `for` instead of `if` in the list comprehension.